axum-extra/src/routing/mod.rs

```diff
@@ -44,6 +167,94 @@ where
     {
         self.merge(routes.routes())
     }
+
+    #[cfg(feature = "typed-routing")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "typed-routing")))]
```

Comment:
If you activate `doc_auto_cfg` instead of `doc_cfg`, `doc(cfg)` attributes are implied and you only need to write them out if you want the docs show some other cfg expression.

Reply:
How do you do that?

I tried setting

```
#![cfg_attr(docsrs, feature(doc_auto_cfg, doc_cfg))]
```

and doing the standard cfg feature

```
#[cfg(feature = "typed-routing")]
pub use axum_macros::TypedPath;
```

and building the docs with

```
RUSTFLAGS="--cfg docsrs" cargo +nightly doc --no-deps
```

But the labels don't show up.

Reply:
maybe one has to use `RUSTDOCFLAGS` instead :think

Reply:
We do basically that (though `doc_cfg` isn't used) in Ruma and it works fine. Can try in axum myself later today.

Reply:
Yep that did it. 8ddf49501fff6

I'll update the rest of axum in a separate PR.